Cargo.toml

```diff
@@ -131,7 +132,7 @@ rand = "0.8"
 redis = { version = "0.25.2", features = ["tokio-rustls-comp", "keep-alive"] }
 regex = "1.10.2"
 reqwest = { version = "0.12", default-features = false, features = ["rustls-tls"] }
-reqwest-tracing = { version = "0.5", features = ["opentelemetry_0_20"] }
+reqwest-tracing = { version = "0.5", features = ["opentelemetry_0_24"] }
```

Comment:
Should this be `opentelemetry_0_25`? Looking Cargo.lock, we're now getting two versions of the opentelemetry crates, 0.24.0 and 0.25.0.

Reply:
While not needed I want to wait for an open PR to reqwest-tracing to add 0.25 support. I will change this PR then and mark it ready. I'm gonna go ping that PR. It's been open for 3w now.

Reply:
On the other hand, it's much easier to go later from 0.24 to 0.25 and dedup the crates for now until reqwest-tracing support for 0.25 landed. So I downgraded to 0.24.

Reply:
ok